doc/lib_enum.md

```diff
@@ -0,0 +1,139 @@
+// This file is distributed under the terms of the MIT license, (c) the KSLib team
+## lib_enum
+
+``lib_enum.ks`` provides a set of functions for manipulating lists using the new delegates syntax.
+
+#### enum_all(<list>, <delegate>) => <boolean>
+
+```
+function is_even { parameter n. return mod(n,2) = 0. }
+enum_all(list(2,4,5), is_even@) => false
+enum_all(list(2,4,6), is_even@) => true
+```
```

Comment:
Yehp, docs written and copied below. The PR should be ready to go on kOS release :)